tests/test_job.py

```diff
@@ -75,6 +75,20 @@ def test_custom_name(hq_env: HqEnv, tmp_path):
     assert len(table) == 2
 
 
+def test_custom_working_dir(hq_env: HqEnv, tmp_path):
+    hq_env.start_server()
+    cwd_custom_tbl = hq_env.command(
+        [
+            "submit",
+            "sleep 1",
+            "--cwd=/test/directory/",
+        ], as_table=True
+    )
+    time.sleep(0.2)
+
+    assert (cwd_custom_tbl[9][1] == "/test/directory/")
```

Comment:
I am not sure how to test the default case here without the --cwd, how do I execute echo $PWD?

Reply:
submit command ``bash -c "echo $PWD"``

Reply:
In terminal, you can run ` ./target/debug/hq submit --cwd /tmp 'echo $PWD'`. This will not unroll `$PWD`. In tests it should not be unrolled automatically I think.

BTW, when I run e.g. ` ./target/debug/hq submit --cwd /tmp echo hello`, the job fails and nothing gets printed to stdout/stderr. Can you please check what is happening?